.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature request
+about: Suggest an idea for the connector
+title: ''
+labels: feature-request
+assignees: ''
+
+---
+
+### Summary
+<!-- Briefly describe the feature in one or two sentences. You can include more details later. -->
+
+### Use cases
+<!-- Describe the use cases that make this feature useful to others.
+The description should help the reader understand why the feature is necessary.
+The better we understand your use case, the better we can help create an appropriate solution. -->
+
+
+### How should it work?
```

Comment:
What about this suggestion? It's maybe a little more in line with what we describe below.
```suggestion
### Proposed solution
```

Reply:
Agreed :) I'll make that change.